src/Zicht/Itertools/twig/Extension.php

```diff
@@ -9,11 +9,6 @@
 
 /**
  * Twig extension.
- *
- * <service id="zicht_itertools_twig_extension" class="Zicht\Itertools\twig\Extension">
- *    <tag name="twig.extension"/>
- * </service>
- *
```

Comment:
:thinking: Why remove this, having it in the readme is good, but any particular reason for removing it from this location?

Reply:
Not particularly, but since the README is the place for instructions (in my opinion) it keeps it maintainable when things change. Otherwise you'll need to look in 2 places and instructions could differ after a while.